ospfd/ospf_abr.c

```diff
@@ -510,15 +515,15 @@ void ospf_check_abr_status(struct ospf *ospf)
 	}
 
 	if (IS_DEBUG_OSPF_EVENT) {
-		zlog_debug("ospf_check_abr_status(): looked through areas");
-		zlog_debug("ospf_check_abr_status(): bb_configured: %d",
-			   bb_configured);
-		zlog_debug("ospf_check_abr_status(): bb_act_attached: %d",
-			   bb_act_attached);
-		zlog_debug("ospf_check_abr_status(): areas_configured: %d",
-			   areas_configured);
-		zlog_debug("ospf_check_abr_status(): areas_act_attached: %d",
-			   areas_act_attached);
+		zlog_debug("%s: looked through areas", __func__);
```

Comment:
IMO, these zlog_debug could be squeezed into a single one?

Reply:
thanks for the review.
In this PR I targeted to clear this __func__ related things.
Next plan I will consider your suggestion.
Thank You.